data/json/recipes/recipe_armor.json

```diff
@@ -8423,8 +8470,7 @@
     [
       [ "ragpouch", 1 ],
       [ "leather_pouch", 1 ],
-      [ "dump_pouch", 1 ],
-      [ "legrig", 1 ]
+      [ "dump_pouch", 1 ]
```

Comment:
Because the leg rig is already a component.
Using two leg rigs is just kinda cheesy (though I suppose you CAN wear two of them with no extra encumbrance).